stix_shifter_modules/guardium/stix_translation/data_mapper.py

```diff
@@ -0,0 +1,13 @@
+from os import path
```

Comment:
I don't think data_mapper.py exists anymore in the latest develop and also its not used in the translation module as well.